templates/detailItems/url.html

```diff
@@ -6,4 +6,4 @@
 	   focus-expression="$parent.ctrl.focus === ctrl.name"
 />
 <button ng-click="ctrl.deleteField()" class="icon-delete" title="Delete"></button>
-<a class="url-link" target="_blank" href="{{ctrl.data.value}}">↗</a>
+<a ng-if="ctrl.data.value.length" class="url-link" target="_blank" href="{{ctrl.data.value}}">↗</a>
```

Comment:
Angular automatically sets `ng-invalid` and `ng-invalid-url` classes. We could use them and only show the link if the url is valid. This could be done in CSS.

BTW: we should use ng-href